### PR TITLE
github issues fixes

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
A PR that solves some of the Github issues

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/constants.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ Elrond uses some constants, which are specific to each chain (mainnet, testnet o
 - https://devnet-gateway.elrond.com/network/config
 - https://github.com/ElrondNetwork/elrond-config-devnet
 
+:::important
+Each transaction requires a `chainID` field that represents the network's identifier. It adds protection so that transactions cannot be replayed from a network to another.
```
Comment (andreibancioiu) [inline docs/developers/constants.md:21]: :+1:

File: docs/tokens/esdt-tokens.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ IssuanceTransaction {
 The receiver address `erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u` is a built-in system smart contract (not a VM-executable contract), which only handles token issuance and other token management operations, and does not handle any transfers.
 The contract will add a random string to the ticker thus creating the **token identifier**. The random string starts with “-” and has 6 more random characters. For example, a token identifier could look like _ALC-6258d2_.
 
+:::note
+Because of the 6 random characters sequence, the token identifier cannot be precalculated, thus one has to check the Smart Contract Result of the issue transaction that indicates it. Alternatively, one can check its tokens via Explorer or API and search for the token that starts with the ticker that was chosen when issuing the token.
```
Comment (andreibancioiu) [inline docs/tokens/esdt-tokens.md:66]: 6 in hex encoding, but 3 bytes, right?

File: docs/validators/key-management/validator-keys.md [hunk 3]
```diff
@@ -36,3 +38,18 @@ More clearly:
 Always save and protect **private keys**, they are like your username + password + 2FA at your bank, all combined.
 
 _Public keys_ are like your phone number - no harm in others knowing it, it actually is needed for some scenarios. Still, only share it on a need to basis, like you would do with your own phone number.
+
+## How to generate a new key
```
Comment (andreibancioiu) [inline docs/validators/key-management/validator-keys.md:42]: :+1:

**Review by andreibancioiu — APPROVED** [review]

File: docs/tokens/esdt-tokens.md (hunk 2 above)
Comment (bogdan-rosianu) [reply docs/tokens/esdt-tokens.md]: extended

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**